src/components/Checkbox/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
Please merge dev into this branch and run lint:fix, it will reformat to match the prettier rules